### PR TITLE
Add connstant for Chef Server and improve help messaging

### PR DESCRIPTION
**Pull request by tas50**
We talk about the server in messages and we need to brand that appropriately.

In the help:

- Periods after each sentence
- Use the branding better
- Fix some weird capitalization

The final bit is the removal of a link to the RFC docs since those are in the boneyard now.

### CODE REVIEW

**Comment by tas50** [comment]
There's some specs to update here.

File: lib/chef/application/client.rb [hunk 1]
```diff
@@ -238,26 +238,26 @@ class Chef::Application::Client < Chef::Application
...
 
   option :chef_zero_host,
     long: "--chef-zero-host HOST",
-    description: "Host to start chef-zero on"
+    description: "Host to start chef-zero on."
 
   option :chef_zero_port,
     long: "--chef-zero-port PORT",
-    description: "Port (or port range) to start chef-zero on.  Port ranges like 1000,1010 or 8889-9999 will try all given ports until one works."
+    description: "Port (or port range) to start chef-zero on. Port ranges like 1000,1010 or 8889-9999 will try all given ports until one works."
```
Comment (mjingle) [inline lib/chef/application/client.rb]:
```suggestion
    description: "Port (or port range) to start chef-zero on. Port ranges like 1000, 1010 or 8889-9999 will try all given ports until one works."
```

File: lib/chef/application/knife.rb [hunk 2]
```diff
@@ -46,112 +46,112 @@ class Chef::Application::Knife < Chef::Application
...
 
   option :chef_zero_host,
     long: "--chef-zero-host HOST",
-    description: "Host to start chef-zero on"
+    description: "Host to start chef-zero on."
 
   option :chef_zero_port,
     long: "--chef-zero-port PORT",
-    description: "Port (or port range) to start chef-zero on.  Port ranges like 1000,1010 or 8889-9999 will try all given ports until one works."
+    description: "Port (or port range) to start chef-zero on. Port ranges like 1000,1010 or 8889-9999 will try all given ports until one works."
```
Comment (mjingle) [inline lib/chef/application/knife.rb]:
```suggestion
    description: "Port (or port range) to start chef-zero on. Port ranges like 1000, 1010, or 8889-9999 will try all given ports until one works."
```

**Comment by mjingle** [comment]
LGTM with the exception of the two suggestions I found.

File: lib/chef/application/client.rb (hunk 1 above)
Comment (tas50) [reply lib/chef/application/client.rb]: I think we need the original formatting actually

**Pull request merged**